test/ex_machina_test.exs

```diff
@@ -103,6 +117,21 @@ defmodule ExMachinaTest do
     end
   end
 
+  test "fields_for/2 returns the fields for the matching factory" do
+    assert MyApp.ExMachina.fields_for(:user) == %{
+      id: 3,
+      admin: false,
+      name: "John Doe"
+    }
+  end
+
+  test "fields_for/2 removes undesireable fields from structs" do
```

Comment:
`undeseriable Ecto fields` to make it clear that's specific to Ecto in this case?